_data/glossary/resource-quota.yaml

```diff
@@ -0,0 +1,11 @@
+id: resource-quota
+name: Resource Quotas
+full-link: /docs/concepts/policy/resource-quotas/
+tags:
+- fundamental
+- operation
+- architecture
+short-description: >
+  Provides constraints that limit aggregate resource consumption per {% glossary_tooltip term_id="namespace" %}.
+long-description: >
+  Limits the quantity of objects that can be created in a namespace by type, as well as the total amount of compute resources that may be consumed by resources in that project.
```

Comment:
s/quantity/number
Also recommend making second clause another sentence, so instead of `as well as ...`, something like:
`Also limits the total amount of compute resources ...`